charts/ssi-dim-wallet-stub/README.md

```diff
@@ -0,0 +1,76 @@
+## Helm char to deploy SSI DIM Wallet stub application
```

Comment:
Typo in chart

Reply:
Fixed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/69c58f27521831fc749979f9248d4ae836418a7e